### DISCUSSION TITLE
Waypoint manipulations

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20042] [Ideas]
OsmAnd has become too bulky an application. I think it is impossible that it serves both navigation and horse routes.
The application should be divided into two applications
![Screenshot_2024-06-09-12-36-29-899_net osmand plus](https://github.com/osmandapp/OsmAnd/assets/53188843/6c12d5ff-8986-43e3-b9bf-bb2b87367241)
In this heat, I'm making a route in an overheated car. Instead of having an overview of the route with waypoints, it shows me the CO2 emissions, and it interferes with my route planning. I know what I need to do to see, but I do it all with one hand with all 4 blinkers on. I'm honestly not interested in CO2 at this point.

There are many such examples in the application

**Comment by 0709wiwiwi** [comment]
Horses also do produce CO2. From 2035 on only electric horses are allowed ;-)

**Comment by ZeljkoBG** [comment]
It's really awkward with waypoint manipulations. This diagram (CO2) to be removed from the display. And to put it as part of the 'Information about the route' or 'Details'.

Why is this only bothering me and @0709wiwiwi?
